docs/component_conventions.md

```diff
@@ -251,6 +251,47 @@ Print styles should be included in the main stylesheet for a component, using th
 }
 ```
 
+### Print style conventions
+
+- Ensure that colours are reset to black using the print specific variable `$govuk-print-text-colour`, for example:
+  - Text: `color: $govuk-print-text-colour;`
+  - Borders: `border-color: $govuk-print-text-colour;`
+- Some elements such as background images do not print by default in most browsers. To force these to print as intended, use the CSS property `print-color-adjust`, for example:
```

Comment:
I don't think this all needs to be in a list - should `Ensure that...` and `Some elements..` just be paragraphs?

Reply:
I've removed the top level list, the text/borders examples remain as a list which is clearer to read.